### ISSUE TITLE
twig/cache-extension is abandoned

### ISSUE THREAD

**Issue by ramunasd** [EmanueleMinotto/TwigCacheBundle#29]
Composer throws warning:
`Package twig/cache-extension is abandoned, you should avoid using it. Use twig/cache-extra instead.`

**Comment by toooni** [comment]
If you are creating a new project you should probably do that. I have not looked into it yet but it seems that this bundle can be deprecated.

**Comment by ramunasd** [comment]
@toooni This warning comes up because that package is abandoned. Development was moved to https://github.com/twigphp/cache-extra
Maybe it's possible to switch dependency easily?

**Comment by toooni** [comment]
@ramunasd Yes, I know. Cache-extra seems very different. Check out https://github.com/twigphp/twig-extra-bundle.
